Empty Maven property tag causes recipe execution failure in `UpdateMavenModel`

**What**
When running a recipe updating the POM file, if an empty property tag is found an java.lang.IllegalStateException is thrown and the recipe is not run.

Since rewrite:7.34.0 (rewrite-maven-plugin:4.38.0) : https://github.com/openrewrite/rewrite/commit/0cd92dadf58b7dcba32b8a9a10a9473b7c188dee

**Steps to reproduce**
- Init a sample java project with maven archetype
`mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=my-app -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false`

- Update App.java like this
```
package com.mycompany.app;

import java.util.concurrent.Callable;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
        System.out.println( "Hello World!" );
        Callable<Integer> callable;
    }
}
```

- Add rewrite.yml at root : 
```
---
type: specs.openrewrite.org/v1beta/recipe
name: covea.jee.core.tools.rewrite.AddPom
displayName: Add Maven dependency example
recipeList:
    - org.openrewrite.maven.AddDependency:
          groupId: com.github.javaparser
          artifactId: javaparser-symbol-solver-core
          onlyIfUsing: java.util.concurrent.Callable
          version: 3.24.4
```

- Run plugin 
`mvn rewrite:run`

Output : 
```

[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------< com.mycompany.app:openrewrite-empty >-----------------
[INFO] Building openrewrite-empty 1.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] >>> rewrite-maven-plugin:4.38.2:run (default-cli) > compile @ openrewrite-empty >>>
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ openrewrite-empty ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory D:\outilsdev\wks\openrewrite-test-bug\openrewrite-empty\src\main\resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ openrewrite-empty ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] <<< rewrite-maven-plugin:4.38.2:run (default-cli) < compile @ openrewrite-empty <<<
[INFO]
[INFO]
[INFO] --- rewrite-maven-plugin:4.38.2:run (default-cli) @ openrewrite-empty ---
Downloading from m2-covea-mirror: https://pic-mirror/bin/simple/m2-mma-group/io/cucumber/messages/maven-metadata.xml
Downloaded from m2-covea-mirror: https://pic-mirror/bin/simple/m2-mma-group/io/cucumber/messages/maven-metadata.xml (2.4 kB at 2.7 kB/s)
[INFO] Using active recipe(s) [covea.jee.core.tools.rewrite.AddPom]
[INFO] Using active styles(s) []
[INFO] Validating active recipes...
[INFO] Project [openrewrite-empty] Resolving Poms...
[INFO] Project [openrewrite-empty] Parsing Source Files
[INFO] Running recipe(s)...
[WARNING] java.lang.IllegalStateException: Maven property tag must have String value
[ERROR] The recipe produced an error. Please report this to the recipe author.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  13.559 s
[INFO] Finished at: 2023-01-11T13:14:11+01:00
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "dev-profile" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.openrewrite.maven:rewrite-maven-plugin:4.38.2:run (default-cli) on project openrewrite-empty: Execution default-cli of goal org.openrewrite.maven:rewrite-maven-plugin:4.38.2:run failed: java.lang.I
llegalStateException: Maven property tag must have String value -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Expected**

Dependency added to POM. 
As show in the referenced commit this behaviour seems intentional, but to me an empty property tag should be allowed as declared in the Maven POM XSD : https://maven.apache.org/xsd/maven-4.0.0.xsd.

--- a/rewrite-maven/src/main/java/org/openrewrite/maven/UpdateMavenModel.java
+++ b/rewrite-maven/src/main/java/org/openrewrite/maven/UpdateMavenModel.java
@@ -43,7 +43,7 @@ public class UpdateMavenModel<P> extends MavenVisitor<P> {
         if (properties.isPresent()) {
             for (final Xml.Tag propertyTag : properties.get().getChildren()) {
                 requested.getProperties().put(propertyTag.getName(),
-                        propertyTag.getValue().orElseThrow(() -> new IllegalStateException("Maven property tag must have String value")));
+                        propertyTag.getValue().orElse(""));
             }
         }
 

